app/Http/Controllers/Api/StatuslabelsController.php

```diff
@@ -30,6 +30,20 @@ public function index(Request $request)
             $statuslabels = $statuslabels->TextSearch($request->input('search'));
         }
 
+
+        // if a status_type is passed, filter by that
+        if ($request->filled('status_type')) {
+            if (strtolower($request->input('status_type'))== 'pending') {
+                $statuslabels = $statuslabels->Pending();
+            } elseif (strtolower($request->input('status_type'))== 'archived') {
+                $statuslabels = $statuslabels->Archived();
+            } elseif (strtolower($request->input('status_type'))== 'deployable') {
+                $statuslabels = $statuslabels->Deployable();
+            } elseif (strtolower($request->input('status_type'))== 'undeployable') {
+                $statuslabels = $statuslabels->Undeployable();
+            }
```

Comment:
I dunno if you care but you don't handle if I pass `?status_type=frooptypoops`.

There's also a way you could have an array of:

```php
$statuses = [
  'pending' => 'Pending',
  'archived' => 'Archived'
  //....
];
```

But that's probably not worth the fact that it's harder to read.

Reply:
Would it make sense to allow a `status_type=all`  (which just adds no query builders to the query?) I'll gladly accept a 'no' to that.

Reply:
I'd say no - if you don't specify, it doesn't try to filter it